snapcraft.yaml

```diff
@@ -42,8 +43,14 @@ parts:
       - make
       - kpartx
       - qemu-utils
+    build-attributes: [ enable-patchelf ]
     override-build: |
-      snapcraftctl build
+      ins_bin=$SNAPCRAFT_PART_INSTALL/bin/
+      mkdir -p "$ins_bin"
+      # Make ubuntu-image statically compiled to avoid libc deps.
+      # We need this as setting interpreter/rpath is known to cause
+      # issues with go binaries (https://github.com/NixOS/patchelf/issues/146).
+      CGO_ENABLED=0 go build -ldflags="-extldflags=-static" -o "$ins_bin" ./cmd/ubuntu-image/
```

Comment:
How come the external linker is being invoked at all if CGO_ENABLED=0? Does ubuntu-image use any of the stdlib code that is disabled without cgo? I don't *think* so but it's worth thinking about (does ubuntu-image do any name resolution itself?)